### ISSUE TITLE
[Solved] torch.nn.CrossEntropyLoss Numerical Error?

### ISSUE THREAD

**Issue by Jiaming-Liu** [pytorch/pytorch#1429]
~~In my own network (2 Conv2D + 3 FC) it consistently gets 1~2% lower acc than this [implement of CrossEntropy+Softmax](https://github.com/Jiaming-Liu/pytorch-misc/blob/master/softmax_with_cross_entropy_loss.py) as well as Keras. Can't reproduce on shallower networks.~~

Actually that implement returns 2*grad of the orig. implement.
